### DISCUSSION TITLE
Use SKIP for multiple headings

### DISCUSSION THREAD

**Discussion by djmtype** [remarkjs/remark#1296] [Q&A] [answered]
Using remark-toc, it seems the `skip` option only accepts a string, but what if you have multiple headings by name you want to skip? 

This does not work: 

```
[remarkToc, {
  skip: ['Fruit', 'Vegetables', 'HABA']
}]
```

**Comment by ChristianMurphy** [comment, marked as answer]
Welcome @djmtype! 👋 
I'd highly recommend reading the readme https://github.com/remarkjs/remark-toc?tab=readme-ov-file#options
Skip is not a string, it is a regular expression, you can add as many strings as you'd like separated by the pipe character.
```js
skip: 'Fruit|Vegetables|HABA'
```

You can learn more about regular expressions here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions

**Reply by djmtype** [reply]
Thanks @ChristianMurphy
